### PR TITLE
Basic skeleton for the flask app

### PR DESCRIPTION
**Pull request by MartinBasti**
Signed-off-by: Martin Bašti <mbasti@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
...
+# Operators Manifests Push Service (OMPS)
+Service for pushing operators manifests to quay.io from various sources.
+
+## Development
+### Installing with test dependencies
+
+To install test dependencies from local directory use following:
+```bash
+pip install '.[test]'
+```
```
Comment (athos-ribeiro) [inline README.md]: It would be nice to include a line on how to run the app in development mode

`FLASK_APP=omps/app.py flask run`

Comment (MartinBasti) [reply README.md]: I plan to use `gunicorn` maybe that can be in example

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='omps',
+    version='0.1',
```
[the remark this answers is not in the document]
Comment (MartinBasti) [reply setup.py]: I can add it there

**Comment by MartinBasti** [comment]
Comments addressed

**Pull request merged**